packages/adapters/wagmi/src/client.ts

```diff
@@ -974,11 +975,7 @@ export class WagmiAdapter implements ChainAdapter {
     bypassWindowCheck = false
   ) {
     if (bypassWindowCheck || (typeof window !== 'undefined' && connector)) {
-      this.appKit?.setLoading(true)
       const provider = (await connector.getProvider()) as W3mFrameProvider
-      const isLoginEmailUsed = provider.getLoginEmailUsed()
-
-      this.appKit?.setLoading(isLoginEmailUsed)
```

Comment:
We don't need to set loading true or false here since we can already do that in `this.syncAccount` when wagmi gives us back status of `'reconnecting'`